### DISCUSSION TITLE
on selectOneMenu the display of long texts is strange.

### DISCUSSION THREAD

**Discussion by spierjoe** [primefaces/community#995] [PrimeFaces] [answered]
After switching to a new primeface version, on selectOneMenu the display of long texts is strange.

```
<p:selectOneMenu id="art" 
	value="#{nkNewRechtsverletzungBean.newInfringementTypeCode}"
	required="true" label="Rechtsverletzung - Typ" style="width:600px;">  
	<f:selectItems value="#{nkNewRechtsverletzungBean.getNKWerteliste('InfringementTypeCode',false,0)}" />  
</p:selectOneMenu> 

```
before:
PrimeFaces 6.2., Extensions 6.2.11
Java SE 8

![Bildschirmfoto 2024-01-22 um 15 05 48](https://github.com/primefaces/community/assets/86762699/47e9a04f-07db-472a-beb7-f2e7c0199f56)

![Bildschirmfoto 2024-01-22 um 15 06 00](https://github.com/primefaces/community/assets/86762699/d6bf27fe-c185-477e-877c-effa1bf5cdc9)

new:
PrimeFaces 13.0.5-jakarta, Extensions 13.0.1-jakarta
jakarta-faces 3.0.5
Java SE 17

![Bildschirmfoto 2024-01-22 um 15 04 47](https://github.com/primefaces/community/assets/86762699/0e40f5c4-d962-4726-a6ad-eeecac4ab0a2)

![Bildschirmfoto 2024-01-22 um 15 04 58](https://github.com/primefaces/community/assets/86762699/eef12d76-487e-4a82-a0bc-a065d88993d7)

**Comment by melloware** [comment]
What theme are you using?  And if you are using a theme did you update to PrimeFaces 13 version?  This looks like a CSS issue possible or an `autoWidth` issue: https://primefaces.github.io/primefaces/13_0_0/#/../migrationguide/13_0_0?id=selectonemenu

**Reply by spierjoe** [reply]
I don't use a theme. I also didn't set any css.
autoWidth is not set.
If I set autoWidth="true" or "false" there is no improvement.
I added the class "ui-fluid", the display is a bit better.

The display is now ok.
![Bildschirmfoto 2024-01-23 um 08 56 13](https://github.com/primefaces/community/assets/86762699/0eccc699-e20e-4380-8c50-d6b450e76146)

But when you open it, everything is still displayed incorrectly.
![Bildschirmfoto 2024-01-22 um 15 04 58](https://github.com/primefaces/community/assets/86762699/ac521a8d-ec9d-4962-b4ac-d9d60ed3343e)

**Reply by melloware** [reply, marked as answer]
@spierjoe here you go: https://github.com/primefaces/primefaces/issues/6877
